### ISSUE TITLE
[Bug]: [Nightly] Get replicas failed when loading completed

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#21025]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: f8cff79
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.3.0.dev15
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Get replicas failed because collection not loaded.
```
[2022-12-06T19:24:18.500Z] [2022-12-06 18:00:24 - ERROR - pymilvus.decorators]: RPC error: [get_replicas], <MilvusException: (code=15, message=failed to get replica info, err=failed to get channels, collection not loaded[CollectionNotFound])>, <Time:{'RPC start': '2022-12-06 18:00:24.346042', 'RPC error': '2022-12-06 18:00:24.349373'}> (decorators.py:108)
[2022-12-06T19:24:18.500Z] [2022-12-06 18:00:24 - ERROR - ci_test]: Traceback (most recent call last):
[2022-12-06T19:24:18.500Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
[2022-12-06T19:24:18.500Z]     res = func(*args, **_kwargs)
[2022-12-06T19:24:18.500Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
[2022-12-06T19:24:18.500Z]     return func(*arg, **kwargs)
[2022-12-06T19:24:18.500Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/collection.py", line 1084, in get_replicas
[2022-12-06T19:24:18.500Z]     return conn.get_replicas(self.name, timeout=timeout, **kwargs)
[2022-12-06T19:24:18.500Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 109, in handler
[2022-12-06T19:24:18.501Z]     raise e
[2022-12-06T19:24:18.501Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 105, in handler
[2022-12-06T19:24:18.501Z]     return func(*args, **kwargs)
[2022-12-06T19:24:18.501Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 136, in handler
[2022-12-06T19:24:18.501Z]     ret = func(self, *args, **kwargs)
[2022-12-06T19:24:18.501Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 85, in handler
[2022-12-06T19:24:18.501Z]     raise e
[2022-12-06T19:24:18.501Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 50, in handler
[2022-12-06T19:24:18.501Z]     return func(self, *args, **kwargs)
[2022-12-06T19:24:18.501Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 1011, in get_replicas
[2022-12-06T19:24:18.501Z]     raise MilvusException(response.status.error_code, response.status.reason)
[2022-12-06T19:24:18.501Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=15, message=failed to get replica info, err=failed to get channels, collection not loaded[CollectionNotFound])>
[2022-12-06T19:24:18.501Z]  (api_request.py:39)
[2022-12-06T19:24:18.501Z] [2022-12-06 18:00:24 - ERROR - ci_test]: (api_response) : <MilvusException: (code=15, message=failed to get replica info, err=failed to get channels, collection not loaded[CollectionNotFound])> (api_request.py:40)
```

### Expected Behavior

pass

### Steps To Reproduce

```markdown
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/212/pipeline/155
```

### Milvus Log

1. [artifacts-milvus-distributed-kafka-nightly-212-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/master/212/artifact/artifacts-milvus-distributed-kafka-nightly-212-pymilvus-e2e-logs.tar.gz)
2. collection name: collection_IFAhpUeT
3. failed time: [2022-12-06T18:00:24.784Z] [gw1] [ 10%] FAILED testcases/test_collection.py::TestLoadCollection::test_load_replica_multi

**Comment by yanliang567** [comment]
@NicoYuan1986 so do you need to load the collection first in the testcase?

**Comment by yanliang567** [comment]
looking into the case, it fails to get the replicas, even after loading

/assign @jiaoew1991 
/unassign

**Comment by NicoYuan1986** [comment]
Some cases related to 'replica' failed too. I think it may help.
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/213/pipeline/155
test_partition_replicas_change_cross_partitions
test_loading_progress_multi_replicas
test_load_replica_multi
test_load_replica_change

**Comment by zhuwenxing** [comment]
It was also reproduced in the upgrade test

error log:
```
2022-12-07T10:25:57.911Z] collection name: task_3_IVF_FLAT

[2022-12-07T10:25:57.911Z] load collection

[2022-12-07T10:25:57.911Z] release collection before load if replicas > 1

[2022-12-07T10:25:57.911Z] release time: 0.0011

[2022-12-07T10:25:57.911Z] load time: 2.0245

[2022-12-07T10:25:57.911Z] Traceback (most recent call last):

[2022-12-07T10:25:57.911Z]   File "scripts/action_after_upgrade.py", line 110, in <module>

[2022-12-07T10:25:57.911Z]     task_3(data_size, host)

[2022-12-07T10:25:57.911Z]   File "scripts/action_after_upgrade.py", line 58, in task_3

[2022-12-07T10:25:57.911Z]     load_and_search(prefix, replicas=NUM_REPLICAS)

[2022-12-07T10:25:57.911Z]   File "/home/jenkins/agent/workspace/tests/python_client/deploy/scripts/utils.py", line 183, in load_and_search

[2022-12-07T10:25:57.911Z]     print(c.get_replicas())

[2022-12-07T10:25:57.911Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/orm/collection.py", line 1084, in get_replicas

[2022-12-07T10:25:57.911Z]     return conn.get_replicas(self.name, timeout=timeout, **kwargs)

[2022-12-07T10:25:57.911Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 109, in handler

[2022-12-07T10:25:57.911Z]     raise e

[2022-12-07T10:25:57.911Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 105, in handler

[2022-12-07T10:25:57.911Z]     return func(*args, **kwargs)

[2022-12-07T10:25:57.911Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 136, in handler

[2022-12-07T10:25:57.911Z]     ret = func(self, *args, **kwargs)

[2022-12-07T10:25:57.911Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 85, in handler

[2022-12-07T10:25:57.911Z]     raise e

[2022-12-07T10:25:57.911Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/decorators.py", line 50, in handler

[2022-12-07T10:25:57.911Z]     return func(self, *args, **kwargs)

[2022-12-07T10:25:57.911Z]   File "/usr/local/lib/python3.8/dist-packages/pymilvus/client/grpc_handler.py", line 1011, in get_replicas

[2022-12-07T10:25:57.911Z]     raise MilvusException(response.status.error_code, response.status.reason)

[2022-12-07T10:25:57.911Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=15, message=failed to get replica info, err=failed to get channels, collection not loaded[CollectionNotFound])>
```

deploy task: upgrade milvus mode：cluster old image tag: v2.1.4 new image tag: master-20221207-dee426a7
failed job: https://qa-jenkins.milvus.io/blue/organizations/jenkins/deploy_test_cron/detail/deploy_test_cron/64/pipeline
log:
[artifacts-pulsar-cluster-upgrade-64-server-logs.tar.gz](https://github.com/milvus-io/milvus/files/10192790/artifacts-pulsar-cluster-upgrade-64-server-logs.tar.gz)

[artifacts-pulsar-cluster-upgrade-64-pytest-logs.tar.gz](https://github.com/milvus-io/milvus/files/10192791/artifacts-pulsar-cluster-upgrade-64-pytest-logs.tar.gz)

**Comment by zhuwenxing** [comment]
many cases failed due this issue
![image](https://user-images.githubusercontent.com/12268675/206652208-37656e12-a376-4b87-81c0-c6b477cc378e.png)

**Comment by NicoYuan1986** [comment]
The issue is not reproduced these days.

**Comment by zhuwenxing** [comment]
Not reproduced in `master-20221210-699b63fb`

**Comment by smellthemoon** [comment]
fixed with #21069 
/assign @NicoYuan1986

**Comment by NicoYuan1986** [comment]
Fixed. master: abc2adb
